.github/CONTRIBUTING.md

```diff
@@ -330,15 +331,55 @@ what you wanted by clicking the `View deployment` or `Visit Preview` buttons on
 This will take you to a preview of the documentation changes.
 This preview is created by [Vercel](https://vercel.com/docs/getting-started-with-vercel).
 
-## 🏭 Release Process
+## 📕 Releases & Versioning
 
 As of now, LangChain has an ad hoc release process: releases are cut with high frequency by
-a developer and published to [PyPI](https://pypi.org/project/langchain/).
+a developer and published to [PyPI](https://pypi.org/project/langchain/). 
+The different packages are versioned slightly differently.
+
+### `langchain-core`
+
+`langchain-core` is currently on version `0.1.x`. 
+
+As `langchain-core` contains the base abstractions and runtime for the whole LangChain ecosystem, we will communicate any breaking changes with advance notice and version bumps. The exception for this is anything in `langchain_core.beta`. The reason for `langchain_core.beta` is that given the rate of change of the field, being able to move quickly is still a priority, and this module is our attempt to do so.
+
+Minor version increases will occur for:
+
+- Breaking changes for any public interfaces NOT in `langchain_core.beta`
+
+Patch version increases will occur for:
+
+- Bug fixes
+- New features
+- Any changes to private interfaces
+- Any changes to `langchain_core.beta`
+
+### `langchain`
+
+`langchain` is currently on version `0.0.x`
+
+All changes will be accompanied by a patch version increase. Any changes to public interfaces are nearly always done in a backwards compatible way.
```

Comment:
worth saying "nearly always done in a backwards compatible way and will be communicated ahead of time when they are not backwards compatible"?